src/operator/c_lapack_api.h

```diff
@@ -361,6 +382,26 @@ inline void flip(int m, int n, DType *b, int ldb, DType *a, int lda) {
   MXNET_LAPACK_CWRAP_SYEVD(ssyevd, float)
   MXNET_LAPACK_CWRAP_SYEVD(dsyevd, double)
 
+  #define MXNET_LAPACK_CWRAP_GESVD(func, dtype) \
```

Comment:
Add comment that due to row-major and internal column-major, the arguments are flipped and transposed, and m and n are flipped as well

Reply:
Added